### ISSUE TITLE
BPTL Scanner issue

### ISSUE THREAD

**Issue by wyattks** [episphere/biospecimen#353]
On the BPTL receipt screen, they can only scan in the FedEx label for two receipts to properly receive them in the dashboard.  When they try to scan in a 3rd FedEx tracking number, it appears quickly and then resets the screen.  It does not let them receive the package by scanning in the ID.  They have to leave the screen and return to have it receive properly.  Norma has reset the scanner and conducted tests with it so it is not a scanner issue.

@Davinkjohnson who would you like me to assign this to?  Also we can have BPTL create FedEx labels for the testing.  Just let us know how many would be needed.

**Comment by Davinkjohnson** [comment]
Let's assign this to Warren. If he has time to look into it. (not sure if it truly requires a scanner or if copy/pasting the label text would work too.)

**Comment by HopkinsDC** [comment]
@we-ai Hi Warren, do you have a scanner to test this issue?

**Comment by we-ai** [comment]
No. I don't have a scanner.

**Comment by wyattks** [comment]
@abhinavjonnada82 I just tested this in Dev.  I had one issue. The second FedEx label (275564747863)
I scanned in did not truncate, it scanned in a long number (1001899922280002087400275564747863) and did not receive but the third one was fine. It wasn't an issue with my scanner.  When I went back to the in Transit screen and came back into the receipt screen, when scanned, the FedEx number was correct.
